app/View/Edit.js

```diff
@@ -45,10 +51,20 @@ class EditView extends React.Component {
         EntityStore.removeFailureListener(this.boundedOnFailure);
     }
 
+    hasEntityAndView(entityName) {
```

Comment:
Yes, it's not the only duplicate code part, but it's more readable IMO.
We need to think about a sort of mixin, will search for a good way to handle this.